pkg/noise/read_writer.go

```diff
@@ -20,7 +21,7 @@ const MaxWriteSize = maxPayloadSize
 
 // Frame format: [ len (2 bytes) | auth & nonce (24 bytes) | payload (<= maxPayloadSize bytes) ]
 const (
-	maxFrameSize   = 4096                                 // maximum frame size (4096)
+	maxFrameSize   = 65536                                // maximum frame size (4096)
```

Comment:
I think we should do this in a different PR unless you have tested it with skywire and everything is working correctly. If you have then just change the comment to `// maximum frame size (65536)`

Reply:
Yeah, you're right. Revert it to 4096. Will make other PR for this.